noteable/sql/connection.py

```diff
@@ -73,6 +70,7 @@ def __init__(
 
         # SLQA-centric fields hereon down, to be pushed into SQLA subclass in the future.
         self._engine = sqlalchemy.create_engine(connection_url, **create_engine_kwargs)
+        self._create_engine_kwargs = create_engine_kwargs
```

Comment:
Stash in the Connection just for use in the test suite as proof they made it here,.